images/ctlog/README.md

```diff
@@ -0,0 +1,13 @@
+<!--monopod:start-->
+# ctlog
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/ctlog` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/ctlog/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
```

Comment:
Could be nice to document this image.

Reply:
I think I want to redirect folks to `sigstore-scaffolding` and have a mega README there that covers all the bits.

@priyawadhwa WDYT?

Reply:
I think that makes sense!